packages/api-sdk/src/medical/models/patient.ts

```diff
@@ -8,6 +8,7 @@ export const patientCreateSchema = demographicsSchema
   .merge(
     z.object({
       externalId: z.string().optional(),
+      additionalFacilityIds: z.array(z.string()).optional(),
```

Comment:
To avoid any breaking changes to the API im leaving the facilityId param as required as this will be the "primary" facility that the patient is associated to and then they can add additional ids if they would like. 

The reason we need the "primary" is because when a patient is created we need a single facility when running PD

Reply:
On DQ then they would add the facility ID that they would like to query the docs for on behalf of that facility.

Reply:
I think we should use a single array of facility IDs. Having `facilityId` and `additionalFacilityIds` is a recipe for error - and it's bad ux IMO.